lib/ansible/cli/galaxy.py

```diff
@@ -697,6 +701,7 @@ def server_config_def(section, key, required, option_type):
         cmd_token = GalaxyToken(token=context.CLIARGS['api_key'])
 
         validate_certs = context.CLIARGS['resolved_validate_certs']
+        default_server_timeout = context.CLIARGS['timeout'] if context.CLIARGS['timeout'] is not None else C.GALAXY_SERVER_TIMEOUT
```

Comment:
won't the cliargs timeout always be set since you set it to default to the C. timeout?

Reply:
no, what i mean is that cliargs' `timeout` should either be  `C.GALAXY_SERVER_TIMEOUT` already or the user supplied value, it seems redundant to have this `if` expression here.

Reply:
ah, yes, that is why the add_argument lacks a default .. ok, nothing to see here

Reply:
:see_no_evil: